src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -905,8 +905,9 @@ private async Task UpdatePackageMaxVulnerabilityAsync(PackageIdentity packageIde
         {
             cancellationToken.ThrowIfCancellationRequested();
 
+            // Use ShutdownToken to avoid canceling on search changes, only on VS shutdown.
```

Comment:
The comment assumes prior knowledge, that originally, we were using the refresh token of the PM UI which got cancelled whenever the packages list was refreshed, including when searching. Consider a comment about what is currently in the code, instead:
 
```suggestion
            // Use ShutdownToken to ensure the operation is canceled if it's still running when VS shuts down.
```